styleguide/js/zoolander.js

```diff
@@ -52,3 +52,19 @@ jQuery(document).ready(function() {
 
     $('#footer-accordion').tabCollapse();
 });
+
+/* Set the width of the side navigation to 250px and the left margin of the page content to 250px */
+function openNav() {
+    document.getElementById("navZoolander-container").style.width = "250px";
+    document.getElementById("mainContainer").style.marginLeft = "250px";
+}
+
+/* Set the width of the side navigation to 0 and the left margin of the page content to 0 */
+function closeNav() {
+    document.getElementById("navZoolander-container").style.width = "0";
+    document.getElementById("mainContainer").style.marginLeft = "0";
+}
+
+jQuery (".navZoolander-hasDropdown").hover(function(){
```

Comment:
Do you wants this removed before each jQuery in the whole file? The entire zoolander.js is written that way.